### DISCUSSION TITLE
WebView support

### DISCUSSION THREAD

**Discussion by luxingxiao** [eclipse-theia/theia-cloud#295] [Q&A] [answered]
Are there any plans to implement support for WebView? I tried it at https://theia-cloud.io/. For example, plug-ins that heavily use WebView like GitLens cannot be used.

**Comment by jfaltermeier** [comment, marked as answer]
Thanks your for the report, I've created this ticket to track this: https://github.com/eclipsesource/theia-cloud/issues/296
I plan to do an initial research about the effort on this soon and will update on the ticket.
